src/main/java/Taylor.java

```diff
@@ -0,0 +1,113 @@
+import java.util.*;
+
+public class Taylor {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+
+        final String line = "____________________________________________________________";
+        System.out.println(line);
+        System.out.println("Hello! I'm Taylor");
+        System.out.println("What can I do for you?");
+        System.out.println(line);
+
+        String input = sc.nextLine();
+        List<Task> tasks = new ArrayList<>();
+        while(!input.equals("bye")) {
+            if(input.equals("list")) {
+                System.out.println(line);
+                System.out.println("Here are the tasks in your list:");
+                for(int i = 0; i < tasks.size(); i++) {
```

Comment:
Since tasks.size() is repeated used in all of the loops, perhaps can create a variable to store this value